src/store/model-store.js

```diff
@@ -49,6 +49,16 @@ class ModelStore extends Store {
     getModelByActionId(actionId) {
         return modelsByActionId[actionId];
     }
+
+    deleteModelByAddress(address){
+        let result = readAddress(address);
+        if(result.type !== 'model' ){
+            throw new Error('Address is not an model.');
+        }
+        delete models[result.index];
```

Comment:
:+1:

This sets it to undefined, but doesn't move other models around in the array. Address lookup should still work. :tada: